ArduCopter/Parameters.cpp

```diff
@@ -317,7 +317,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: LOG_BITMASK
     // @DisplayName: Log bitmask
     // @Description: 4 byte bitmap of log types to enable
-    // @Values: 830:Default,894:Default+RCIN,958:Default+IMU,1854:Default+Motors,-6146:NearlyAll-AC315,45054:NearlyAll,131071:All+FastATT,262142:All+MotBatt,393214:All+FastIMU,397310:All+FastIMU+PID,655358:All+FullIMU,0:Disabled
+    // @Values: -6146:NearlyAll-AC315,0:Disabled,830:Default,894:Default+RCIN,958:Default+IMU,1854:Default+Motors,45054:NearlyAll,131071:All+FastATT,262142:All+MotBatt,393214:All+FastIMU,397310:All+FastIMU+PID,655358:All+FullIMU
```

Comment:
This might show the flaw in the PR; people might want the default as teh first thing in the list.  We'll see what happens at the devcall.

Reply:
the negative numbers are an aberration due to the mapping of the values

Reply:
We need a @ defaultValue thing, or an autodetected default. having it as first in the list is not very reliable for people to go by